README.md

```diff
@@ -59,8 +59,8 @@ CF_ROOT_DOMAIN=<the root domain you want to manage. ex. example.com>
 SERVICE=nginx <the name of the service you want to monitor>
 RECORD=<the A-record you want to manage. ex. my.example.com>
 TTL=600 <the DNS TTL you want>
```

Comment:
Actually, the minimum is 120 seconds. I tried to set it to 60 seconds in my first go and got bupkis. Fortunately the error message from CloudFlare explained it.